include/mgba/internal/script/socket.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2013-2022 Jeffrey Pfau
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+#ifndef M_SCRIPT_SOCKERR_H
+#define M_SCRIPT_SOCKERR_H
+
+enum mSocketErrorCode {
+	SOCKERR_UNKNOWN_ERROR = -1,
+	SOCKERR_OK = 0,
+	SOCKERR_AGAIN,
+	SOCKERR_ADDRESS_IN_USE,
+	SOCKERR_CONNECTION_REFUSED,
+	SOCKERR_DENIED,
+	SOCKERR_FAILED,
+	SOCKERR_NETWORK_UNREACHABLE,
+	SOCKERR_NOT_FOUND,
+	SOCKERR_NO_DATA,
+	SOCKERR_OUT_OF_MEMORY,
+	SOCKERR_TIMEOUT,
+	SOCKERR_UNSUPPORTED,
```

Comment:
Though they're internal only so it probably doesn't matter that much, you can take a look at the PLATFORM exports to see how to export a constant that has a different name from its exported namespace.

Reply:
Done